framework/resources/Functional/filter/pushdown/plan/drill_6199_filter_push_down_nested_subqueries_plan.res

```diff
@@ -0,0 +1 @@
+.*Scan.*path=.*/DRILL_6118_parquet_partitioned_by_folders/d1/0_0_0.parquet.*path=.*/DRILL_6118_parquet_partitioned_by_folders/d2/0_0_0.parquet.*numFiles=2.*numRowGroups=2
```

Comment:
Can the order of the two scanned files change for different runs in the plan? If yes, you need to separate them into multiple lines.

Reply:
It works as you say if the verification type is set to "regex-no-order". And I added tests to the existing test suite, which has "regex", so if I change the order, the tests would fail, no matter, how many lines result has. 
I can move the tests into a separate test suite, as a solution. Will it be OK?

Reply:
If the order of the two scanned files can change from run to run, you cannot use "regex".  This is why I added "regex-no-order".  If the new plans need "regex-no-order", then putting them in a separate folder is a good idea.  "regex-no-order" is a weaker verification method than "regex".

Reply:
In general, I do not believe our regex cares how many lines are used in the results file.  The contents of the results file is presented to the verifier as a single long string.  If a pattern is split into multiple lines, there will be additional newline characters, but that is about it.

Reply:
@rhou1 , thanks for clarifying! @agozhiy , I have no objections now.

Reply:
sorry, I was talking about the other PR - where was there a conversation on regex.

Reply:
@rhou1 You are right the baseline is presented to the verifier as a single long string but this long string is actually broken up into smaller pieces based on newline characters and verified piece by piece. Only in this way you can avoid ordering.

@agozhiy Another way to handle filenames without using the "regex-no-order" is to omit filename verification all together. You can rely on number of files in the plan verification and the correctness of the actual query to achieve the same. I had similar conversation with @rhou1 before. But it's your choice here. The framework provides the flexibility to handle either approach.